### PR TITLE
New strings PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -455,6 +457,7 @@
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimo com juros de 0% em seu bitcoin",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:460]:
```suggestion
      "zero": "Empréstimo de dólar com juros de 0% garantidos por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -455,6 +457,7 @@
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimo de dólar com juros de 0% garantidos por seus bitcoins",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:460]:
```suggestion
      "zero": "Empréstimo com juros de 0% garantidos por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -455,6 +457,7 @@
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimo com juros de 0% garantidos por seus bitcoins",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:460]:
```suggestion
      "zero": "Empréstimos com juros de 0% garantidos por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -455,6 +465,7 @@
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimos com juros de 0% garantidos por seus bitcoins",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:468]:
```suggestion
      "zero": "Empréstimos com 0% de juros garantidos por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 5]
```diff
@@ -447,14 +457,15 @@
...
-      "marginTrade": "Negocie com alavancagem de até 10x",
+      "marginTrade": "Negocie com alavancagem de até 5x",
       "lend": "Receba juros emprestando criptos nos pools de empréstimo",
       "pool": "Ganhe rendimentos depositando criptos nos pools de liquidez",
       "staking": "Ganhe recompensas investindo seus SOV",
       "originsLaunchpad": "Vendas realizadas na plataforma Origens",
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimos com 0% de juros garantidos por seus bitcoins",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:468]:
```suggestion
      "zero": "Empréstimos sem juros garantido por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 6]
```diff
@@ -447,14 +457,15 @@
...
-      "marginTrade": "Negocie com alavancagem de até 10x",
+      "marginTrade": "Negocie com alavancagem de até 5x",
       "lend": "Receba juros emprestando criptos nos pools de empréstimo",
       "pool": "Ganhe rendimentos depositando criptos nos pools de liquidez",
       "staking": "Ganhe recompensas investindo seus SOV",
       "originsLaunchpad": "Vendas realizadas na plataforma Origens",
       "originsClaim": "Reivindicar recompensas FISH",
       "myntToken": "Protocolo Sovryn de moedas estáveis apoiado por BTC",
       "perpetuals": "Em breve",
+      "zero": "Empréstimos sem juros garantido por seus bitcoins",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:468]:
```suggestion
      "zero": "Empréstimos sem juros garantidos por seus bitcoins",
```

File: src/locales/pt_br/translation.json [hunk 7]
```diff
@@ -754,6 +793,7 @@
       "totalStaked": "Total investido",
       "viewHistory": "Ver histórico",
       "status": "Status da transação",
+      "emptyHistory": "Hitórico vázio.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:796]:
```suggestion
      "emptyHistory": "Histórico vázio.",
```

File: src/locales/pt_br/translation.json [hunk 8]
```diff
@@ -1240,8 +1298,8 @@
       "tier": "Tier",
       "sovOG": "SOV-OG",
       "btc": "BTC",
-      "emptyGallery": "Your connected wallet does not currently have any supported NFTs. Join the <0>Sovryn community</0> for opportunities to earn NFT rewards.",
-      "loadingMessage": "Loading your gallery."
+      "emptyGallery": "Sua carteira não possui NFTs. Junte-se a <0>comunidade Sovryn</0> para oportunidades de ganhar recompensas NFT.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1301]:
```suggestion
      "emptyGallery": "Sua carteira não possui NFTs. Junte-se à <0>comunidade Sovryn</0> para oportunidades de ganhar recompensas NFT.",
```

File: src/locales/pt_br/translation.json [hunk 9]
```diff
@@ -435,7 +470,7 @@
     "claim": "Reinvindicar",
     "rewards": "Recompensas",
     "labs": "Labs",
-    "myntToken": "MYNT token",
+    "myntToken": "Tokeb MYNT",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:473]:
```suggestion
    "myntToken": "Token MYNT",
```

File: src/locales/pt_br/translation.json [hunk 10]
```diff
@@ -694,6 +732,7 @@
   "stake": {
     "title": "SOV investidos/reservados",
     "meta": "SOV investidos/reservados",
+    "paused": "Staking functionality is currently paused by Sovryn Governance, please try again later or check our <0>Discord</0> for more details.",
     "connect": "Conecte sua carteira e comece a rentabilizar seus SOV.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:736]:
```suggestion
    "connect": "Conecte sua carteira e comece a rentabilizar seus SOVs.",
```

File: src/locales/pt_br/translation.json [hunk 11]
```diff
@@ -1687,117 +1763,123 @@
...
-        "sell": "Venda",
-        "connect": "Conectar Carteira",
-        "market": "Mercado",
-        "limit": "Limitada",
-        "buyMarket": "Comprar a mercado",
-        "buyLimit": "Comprar a limite",
-        "sellMarket": "Vender a mercado",
-        "sellLimit": "Vender a limite"
+      "p3": {
+        "title": "Distribuição de 30K SOV no pool SOV/RBTC!",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1837]:
```suggestion
        "title": "Distribuição de 30K SOVs no pool SOV/RBTC!",
```

File: src/locales/pt_br/translation.json [hunk 12]
```diff
@@ -1687,117 +1763,123 @@
...
+        "title": "Status da transação",
+        "aborted": "Você optou por rejeitar esta transação. Cancele ou refaça a transação",
+        "abortedLedger": "Verifique se \"Contract data\" está configurado para \"Permitido\" no seu aplicativo Ledger",
+        "cta": "Abrir transação"
+      }
+    },
+    "promotions": {
+      "title": "Promoções SOV",
+      "p1": {
+        "title": "Mês de Abril - Distribuição de 50k SOV no pool BTC/USDT",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1827]:
```suggestion
        "title": "Mês de Abril - Distribuição de 50k SOVs no pool BTC/USDT",
```

File: src/locales/pt_br/translation.json [hunk 13]
```diff
@@ -411,15 +446,15 @@
     "swap": "Trocar cripto",
     "marginTrade": "Operar margem",
     "earn": "Ganhe",
-    "pool": "Minerar liquidez",
+    "pool": "Minerar liquidez - Farming",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:449]:
```suggestion
    "pool": "Minerar liquidez",
```

File: src/locales/pt_br/translation.json [hunk 14]
```diff
@@ -411,15 +446,15 @@
...
     "voting": "Votar",
     "portfolio": "Portfólio",
     "finance": "Finanças",
     "lend": "Emprestar cripto",
     "borrow": "Pegar empréstimo",
-    "yieldFarm": "Minerar liquidez",
+    "yieldFarm": "Minerar liquidez - Farming",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:455]:
```suggestion
    "yieldFarm": "Minerar liquidez",
```

File: src/locales/pt_br/translation.json [hunk 15]
```diff
@@ -411,15 +446,15 @@
...
     "reward": "Bônus SOV",
-    "staking": "Rentabilizar SOV",
+    "staking": "Rentabilizar SOV - Staking",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:457]:
```suggestion
    "staking": "Rentabilizar SOV",
```

File: src/locales/pt_br/translation.json [hunk 16]
```diff
@@ -1687,117 +1763,123 @@
...
-        "pair": "Par de negociação:",
-        "asset": "Ativo:",
-        "leverage": "Alavancagem:",
-        "orderValue": "Valor da ordem:",
-        "limitPrice": "Preço máximo:",
-        "slippage": "Diferença máxima no preço:",
-        "balance": "Saldo disponível: <0/> <1/>",
-        "txFee": "Taxa de transação: <0/> {{symbol}}"
+      "p2": {
+        "title": "Mês de Abril - Distribuição de 75k SOV no pool BTC/SOV",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1832]:
```suggestion
        "title": "Mês de Abril - Distribuição de 75k SOVs no pool BTC/SOV",
```

File: src/locales/pt_br/translation.json [hunk 17]
```diff
@@ -2174,133 +2131,604 @@
...
+      "24h": "24h%",
+      "7d": "7d%",
+      "marketCap": "Valor de mercado",
+      "marketCapTooltip": "O valor de mercado é calculado multiplicando o total em circulação pelo preço em dólar. <br /> <br /> Para SOV, o total em circulação é igual a oferta total (100m) menos os tokens reservados, <0>não inclui </0> os tokens de contratos de fundos de adoção e desenvolvimento. <br /> <br /> Para rBTC, o total em circulação é igual ao total de BTC na rede RSK <br /> <br /> Para os demais tokens, o total em circulação é igual a oferta total <br /> <br /> Para os tokens de outras blockchains, o total em circulação é igual ao total existente na rede RSK. Por exemplo, o total em circulação de ETHs não é igual ao total em circulação de ETH na ethereum.",
+      "circulatingSupply": "Em circulação"
+    }
+  },
+  "promotions": {
+    "card1": {
+      "title": "5K SOVs em recompensas",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:2646]:
```suggestion
      "title": "5K SOVs bônus",
```

File: src/locales/pt_br/translation.json [hunk 18]
```diff
@@ -2174,133 +2131,604 @@
...
+      "duration": "Toda semana",
+      "text": "Deposite MYNT e rBTC neste pool de liquidez e comece agora mesmo a acumular sua parte nos 5,000 SOVs."
+    },
+    "card2": {
+      "title": "15K SOVs em recompensas",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:2651]:
```suggestion
      "title": "15K SOVs bônus",
```

File: src/locales/pt_br/translation.json [hunk 19]
```diff
@@ -2174,133 +2131,604 @@
...
+      "duration": "Toda semana",
+      "text": "Deposite qualquer quantia de XUSD neste pool de empréstimo e comece agora mesmo a acumular sua parte nos 15,000 SOVs."
+    },
+    "card3": {
+      "title": "25K SOVs em recompensas",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:2656]:
```suggestion
      "title": "25K SOVs bônus",
```

File: src/locales/pt_br/translation.json [hunk 20]
```diff
@@ -2174,133 +2131,604 @@
...
+      "duration": "Toda semana",
+      "text": "Deposite XUSD e rBTC neste pool de liquidez e comece agora mesmo a acumular sua parte nos 25,000 SOVs."
+    },
+    "card4": {
+      "title": "30K SOVs em recompensas",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:2661]:
```suggestion
      "title": "30K SOVs bônus",
```

File: src/locales/pt_br/translation.json [hunk 21]
```diff
@@ -2174,133 +2131,604 @@
...
+      "duration": "Toda semana",
+      "text": "Deposite SOV e rBTC neste pool de liquidez e comece agora mesmo a acumular sua parte nos 30,000 SOVs."
+    },
+    "card5": {
+      "title": "5K SOVs em recompensas",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:2666]:
```suggestion
      "title": "5K SOVs bônus",
```

File: src/locales/pt_br/translation.json [hunk 22]
```diff
@@ -150,7 +172,7 @@
   "marketingPage": {
     "explain": {
       "title": "COMO FUNCIONA?",
-      "earn": "Ganhe SOV minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
+      "earn": "Ganhe SOVs minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
       "datesUSDT": "Datas USDT/RBTC: 1 Abril 2021, 12.00 UTC  até 1 Maio 2021, 12.00 UTC",
       "datesSOV": "Datas SOV/RBTC: 13 Abril 2021 12.00 UTC até 11 Maio 2021 12.00 UTC",
       "depositUSDT": "Bônus se depositar USDT: 25,000 SOV",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:178]:
```suggestion
      "depositUSDT": "Bônus se depositar USDT: 25,000 SOVs",
```

File: src/locales/pt_br/translation.json [hunk 23]
```diff
@@ -150,10 +172,10 @@
   "marketingPage": {
     "explain": {
       "title": "COMO FUNCIONA?",
-      "earn": "Ganhe SOV minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
+      "earn": "Ganhe SOVs minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
       "datesUSDT": "Datas USDT/RBTC: 1 Abril 2021, 12.00 UTC  até 1 Maio 2021, 12.00 UTC",
       "datesSOV": "Datas SOV/RBTC: 13 Abril 2021 12.00 UTC até 11 Maio 2021 12.00 UTC",
-      "depositUSDT": "Bônus se depositar USDT: 25,000 SOV",
+      "depositUSDT": "Bônus se depositar USDT: 25,000 SOVs",
       "depositRBTC": "Bônus se depositar RBTC: 25,000 SOV",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:179]:
```suggestion
      "depositRBTC": "Bônus se depositar RBTC: 25,000 SOVs",
```

File: src/locales/pt_br/translation.json [hunk 24]
```diff
@@ -150,11 +172,11 @@
...
       "title": "COMO FUNCIONA?",
-      "earn": "Ganhe SOV minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
+      "earn": "Ganhe SOVs minerando liquidez nos pools USDT/BTC ou SOV/RBTC",
       "datesUSDT": "Datas USDT/RBTC: 1 Abril 2021, 12.00 UTC  até 1 Maio 2021, 12.00 UTC",
       "datesSOV": "Datas SOV/RBTC: 13 Abril 2021 12.00 UTC até 11 Maio 2021 12.00 UTC",
-      "depositUSDT": "Bônus se depositar USDT: 25,000 SOV",
-      "depositRBTC": "Bônus se depositar RBTC: 25,000 SOV",
+      "depositUSDT": "Bônus se depositar USDT: 25,000 SOVs",
+      "depositRBTC": "Bônus se depositar RBTC: 25,000 SOVs",
       "depositSOV": "Bônus se depositar SOV/RBTC: 75,000 SOV (15k /por semana mais 15k extra para primeira semana)",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:180]:
```suggestion
      "depositSOV": "Bônus se depositar SOV/RBTC: 75,000 SOVs (15k /por semana mais 15k extra para primeira semana)",
```

File: src/locales/pt_br/translation.json [hunk 25]
```diff
@@ -1306,6 +1365,10 @@
   "lendingPage": {
     "deposit": "Emprestar",
     "withdraw": "Resgatar",
+    "meta": {
+      "title": "Emprestar",
+      "description": "Lending Page"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1370]:
```suggestion
      "description": "Emprestar ativos"
```

File: src/locales/pt_br/translation.json [hunk 26]
```diff
@@ -1306,6 +1365,10 @@
   "lendingPage": {
     "deposit": "Emprestar",
     "withdraw": "Resgatar",
+    "meta": {
+      "title": "Emprestar",
+      "description": "Emprestar ativos"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1370]:
```suggestion
      "description": "Emprestar criptos"
```

File: src/locales/pt_br/translation.json [hunk 27]
```diff
@@ -1195,6 +1251,8 @@
       "redeemRBTC": "Sacar rBTC",
       "withdraw": "Sacar"
     },
+    "sendMessage": "Envie {{asset}} para um endereço BTC",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1254]:
```suggestion
    "sendMessage": "Envie {{asset}}",
```

File: src/locales/pt_br/translation.json [hunk 28]
```diff
@@ -1195,6 +1251,8 @@
...
+    "receiveMessage": "Receba {{asset}} depositando BTC",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1255]:
```suggestion
    "receiveMessage": "Receba {{asset}}",
```

**Comment by soulBit** [comment]
Merged via #2277

Just so you are aware @bitcoinuser, there was an error in the json you submitted - a missing "," after the `claimRewards` property - fixed when merging (https://github.com/DistributedCollective/Sovryn-frontend/pull/2277/commits/54f0247e90fd5639cf7d8d0a37b23f2a3e2356fb#diff-c2d65527dd0b3824aca8eb1cb1e840bb5e7fff30c84a4d4732170d7a69eb9c8bR94), but if possible please run the json file through a format check tool like https://jsonlint.com to make sure it doesn't have similar errors in future

**Pull request closed without merging**